### PR TITLE
Create promo wheel iam html template

### PR DESCRIPTION
**Pull request by Koji23**
Followed https://www.youtube.com/watch?v=aKmR3wbQQWE&ab_channel=codenat

https://user-images.githubusercontent.com/10385763/177842644-3812e383-aaea-41b5-8704-28979489f048.mov

### CODE REVIEW

File: promo_wheel/index.html [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+              "Congratulations";
+            document.querySelector(".secondary-header").innerText =
+              "You unlocked a good discount";
+          }, 5000);
+          }
+        });
+
+        function handlePromoReceived(number) {
+          var finalRotation = number % 360;
+          console.log("number:", number, "finalRotation:", finalRotation);
```
Comment (gardenFiend138) [inline promo_wheel/index.html:369]: do we want to be logging this?

File: promo_wheel/index.html [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+        <input
+          type="submit"
+          value="Try My Luck!"
+          data-onesignal-unique-label="submit-button"
+        />
+      </form>
+    </div>
+    <script type="text/javascript">
+      window.deal = "";
+      window.promoCode = "";
```
Comment (gardenFiend138) [inline promo_wheel/index.html:314]: if we just assign these with `var` won't they be in the global scope vs assigning them directly to the window?

File: promo_wheel/index.html [hunk 3]
```diff
@@ -0,0 +1,407 @@
...
+
+        document.querySelector("form").addEventListener("submit", function (e) {
+          e.preventDefault();
+          e.stopPropagation();
+          var emailAddress = e.target["email-address"].value;
+          var wheel = document.querySelector(".wheel");
+          var number = Math.ceil(Math.random() * 10000);
+          var edges = [30, 90, 150, 210, 270, 330];
+          if (edges.includes(number)) {
+            // Add one so that the wheel never lands on an edge 🤫
```
Comment (gardenFiend138) [inline promo_wheel/index.html:337]: nice

**Review by iAmWillShepherd — APPROVED** [review]
Token approval

File: promo_wheel/index.html (hunk 1 above)
Comment (Koji23) [reply promo_wheel/index.html]: removed

File: promo_wheel/index.html (hunk 2 above)
Comment (Koji23) [reply promo_wheel/index.html:326]: that's true

**Pull request merged**